src/universal/Atmosphere.js

```diff
@@ -70,7 +70,7 @@ export default class Atmosphere extends Environment {
     await this.upgradeTransport()
     const newQuerySubs = subConfigs.map((config) => {
       const {subscription, variables = {}} = config
-      const {name} = subscription()
+      const {name} = getRequest(subscription)
```

Comment:
this prevented additional subscriptions from being registered. worst case scenario is some agenda items or team members would get added to the team & other folks might not see that. it was introduced when we added typescript since all graphql tags turn into commonjs objects instead of ESM objects. getRequest handles both.

Reply:
I tried to figure out what this was doing on a deeper level so I could understand it beyond your comment and came up empty. I couldn't track down any docs for `relay-runtime`'s getRequest. So I decided to just say:

`¯\_(ツ)_/¯`